package.json

```
@@ -7,6 +7,7 @@
     "kuzzle": "./bin/start-kuzzle-server"
   },
   "scripts": {
+    "start:dev": "node -r ts-node/register docker/scripts/start-kuzzle-dev.ts --enable-plugins functional-test-plugin",
```

Good idea for running kuzzle outside docker images. Though it would probably be better to start it with the debugger on, as well as the automated reloader:

```suggestion
    "start:dev": "node docker/scripts/reloader.js --inspect=0.0.0.0:9229 -r ts-node/register docker/scripts/start-kuzzle-dev.ts --enable-plugins functional-test-plugin",
```

wdyt?